src/extension/commandPaletteHandler.ts

```diff
@@ -388,12 +411,24 @@ export class CommandPaletteHandler {
         }
     }
 
+    private static checkReactNativePackageExistence(workspaceRoot: string): Q.Promise<string> {
```

Comment:
Why we need that function? We can use getversionfromnodemodules for that. If it returns version then package exists.